go.mod

```diff
@@ -12,28 +12,25 @@ require (
 	github.com/ONSdigital/dp-net v1.0.11
 	github.com/ONSdigital/go-ns v0.0.0-20200902154605-290c8b5ba5eb
 	github.com/ONSdigital/log.go v1.0.1
-	github.com/Shopify/sarama v1.27.2 // indirect
-	github.com/fatih/color v1.10.0 // indirect
+	github.com/armakuni/dp-go-featuretest v0.0.0-20210211154136-f4f263cb8559
+	github.com/benweissmann/memongo v0.1.1
+	github.com/cucumber/godog v0.10.0
```

Comment:
This library isn't stable as mentioned in their README, it explicitly states that the API is likely to change a few times before reaching its 1.0.0 version. I personally don't know how to act on this but I feel like mentioning it here if any discussion wants to be made on it

Reply:
jon to ask godog authors